nikola/nikola.py

```diff
@@ -1031,7 +1031,7 @@ def init_plugins(self, commands_only=False, load_all=False):
         extra_plugins_dirs = self.config['EXTRA_PLUGINS_DIRS']
         self._loading_commands_only = commands_only
         self._plugin_places = [
-            resource_filename('nikola', 'plugins'),
+            importlib_resources('nikola') / 'plugins',
```

Comment:
1. You can’t call a module like that, this code couldn’t have worked.
2. [We want to use the standard library `importlib.resources`](https://docs.python.org/3/library/importlib.resources.html).